data/Witkowski_1991/metadata.yml

```diff
@@ -0,0 +1,257 @@
+    with 4-5 months for the northern sandplains (Bartlett 1975). Annual rainfall is
+    528 mm (range 376-687 mm), and mean maximum and minimum temperatures are 25.7OC
+    and 7.7OC respectively. Soils are at least 10m deep, infertile, mildly acidic
+    (pH 5.9 5 0.2), siliceous sands
+  original_file: none, transcribed from manuscript
+  notes: none
+locations:
+  Hopetoun:
+    latitude (deg): -33.95
+    longitude (deg): 120.383
+    description: dune system of deep infertile siliceous sands
+    locality: Hopetoun, Western Australia
+    fire history: The site was burnt about 10 years ago by a patchy (probably spring)
+      burn which resulted in intermingled remnants of the parent population which
+      were 21 years old at the time of the study (stem increment counts, Lamont 1985).
+      A second site, 4 k m E. of the first, consisted of a stand of only 21-year-old
+      plants.
+    vegetation type: Banksia scrub-heath about 3 m in height, consisting of evergreen
+      sclerophyllous trees, shrubs and herbs
+    soil type: Soils are at least 10m deep, infertile, mildly acidic (pH 5.9 5 0.2),
+      siliceous sands.
+contexts: .na
+traits:
+- var_in: reproductive_maturity
+  unit_in: a
+  trait_name: reproductive_maturity
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: .na
+  methods: Fifteen young (10 years) and five old (21 years) individuals of each species
+    were chosen at random in autumn 1988 to study their seed bank dynamics. The numbers
+    of fertile (with follicles) and infertile cones produced each year (determined
+    by stem increments, Lamont 1985) were counted. Ten cones (or the total if < 10)
+    of each age were sampled from each plant. Age to first flowering and age to first
+    fertile cone were determined. The number of florets in 20 of the most recent flower
+    heads from the younger plants was counted and relative fruit set and seed set
+    estimated. Masses of 20 seeds from the most recent crop of the 10-year-old plants
+    were determined. Numbers of opened and closed follicles per cone were counted.
+    ^p Cones were burnt to rupture the follicles, immersed in water for 1 min and
+    left to dry for a week to aid seed removal. Each closed follicle contained two
+    seeds. Seeds were classified as aborted, eaten/ decayed, firm or dispersed. The
+    viability of firm seeds was determined by germinating two batches of 30 firm seeds
+    of each age. Some of the older ages were pooled owing to low numbers of seeds
+    available. Seeds were surface sterilised using sodium hypochlorite (5%) for 5min,
+    rinsed and placed in filter paper over vermiculite in petri dishes (150 mm diam.).
+    These were moistened with a fungicid solution (Benlate @, 1 gL-I), and kept in
+    a growth cabinet at 15OC. Germination was observed for 50 days and percentage
+    germination calculated.
+- var_in: seed_germination_%
```

Comment:
Should there be a context variable to describe the environmental/experimental conditions for seed germination? Maybe just a simple one to say cones were burnt? I guess that's not necessarily about the conditions for seed germination but how the seeds were extracted from the follicles.

Reply:
I feel like this is methods - context should be something that is different across rows of data that can't be captured in methods.